trunk/SUAVE/Methods/Aerodynamics/AVL/read_results.py

```diff
@@ -36,35 +36,42 @@ def read_results(avl_object):
     #i = 0 Used in the dynamic stability module (under development)
     for case_name in avl_object.current_status.cases:
         case = avl_object.current_status.cases[case_name]
-        num_ctrl = case.stability_and_control.control_deflections.size
+        num_ctrl =  case.stability_and_control.control_surfaces
```

Comment:
Looks like this needs a different variable name if it is a number.

Reply:
Yeah I saw the redundancies. I changed the variable name. Issue Resolved